README.md

```diff
@@ -1,3 +1,21 @@
 # hackmud_wiki
 
 [![License: CC BY-NC-SA 4.0](https://img.shields.io/badge/License-CC_BY--NC--SA_4.0-lightgrey.svg)](https://creativecommons.org/licenses/by-nc-sa/4.0/)
+
+## Setting up a local environment
+
+### Prerequisites
+
+- [pnpm](https://pnpm.io/installation)
+
+### Installation
+
+1. Clone the repository
+2. Run `pnpm install` in the root directory
+3. Set your Node.js runtime to version >= `20.x` (as specified in `.package.json`)
+    - You can achieve this by running `pnpm env use 20.x --global`
+
+### Running the development server
+
+1. Run `pnpm start` in the root directory
+2. Navigate to `https://localhost:3000/hackmud_wiki`
```

Comment:
the :3000 here may change and need to be updated. Can we change the instructions to reference the cli output?

```[SUCCESS] Docusaurus website is running at: http://localhost:3000/hackmud_wiki/```

like, click the link or allow the cli to open your browser window when it requests it?